x-pack/plugins/maps/ui_tests/fixtures/page_objects/gis_page.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ScoutPage} from '@kbn/scout';
+
+export class GisPage {
+  constructor(private readonly page: ScoutPage) {}
+
+  async fullScreenModeMenuItemExists() {
+    await this.page.testSubj.locator('mapsFullScreenMode').waitFor({ state: 'visible' });
+  }
+
+  async clickFullScreenMode() {
+    await this.page.testSubj.click('mapsFullScreenMode')
+  }
+
+  async clickExitFullScreenTextButton() {
+    await this.page.testSubj.click('exitFullScreenModeText');
+  }
```

Comment:
I'm not convinced we need to wrap single-action like type/click within Page Objects, unless it is extremely repetitive action. 

Looking into `git_page.ts` I think we definetly have to migrate `waitForLayersToLoad` , `loadSavedMap`, `saveMap` and other complex logic that is hard to understand without wrapping function.   

`maps.clickFullScreenMode()` is only used in the **single** test, I'm not sure it needs a wrapper function.

Reply:
🏁